packages/components/src/tab-panel/README.md

```diff
@@ -143,6 +143,14 @@ The name of the tab to be selected upon mounting of component. If this prop is n
 -   Required: No
 -   Default: none
 
+#### hasManualTabActivation
```

Comment:
Not entirely convinced about this new prop's name, open to suggestions!

Reply:
Maybe just `manualActivation`?

Reply:
For boolean props I like to use names that start with `is` or `has`, which makes it more obvious that the prop is a boolean — so maybe `hasManualActivation` ?

Reply:
@chad1008 's [suggestion](https://github.com/WordPress/gutenberg/pull/46004#issuecomment-1325152341) for `selectOnMove` sound good to me.

Reply:
Done!